avd_docs/kubernetes/general/AVD-KSV-0107/docs.md

```diff
@@ -1,10 +1,13 @@
 
-apiVersion and kind has been deprecated
+apiVersion '' and kind '' has been deprecated on: '' and planned for removal on:''
```

Comment:
Is this change part of the PR?

Reply:
Yes, this change is automatically generated by running make docs command